Library/Formula/cabocha.rb

```diff
@@ -35,6 +33,6 @@ def install
     ]
 
     system "./configure", *args
-    system "make install"
+    system "make", "install"
   end
 end
```

Comment:
Can a test be added to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. No worries if not. Thanks!